components/vm/12-interpolated-defaults.tf

```diff
@@ -3,12 +3,25 @@ locals {
     "stg"  = "nle"
     "prod" = "prod"
   }
+
+  xdr_env = var.env == "stg" ? "nonprod" : var.env
+
+  xdr_tags_map = {
+    activityName = "heritage"
+    application  = "crimeportal"
+    org          = "hmcts"
+    server       = "server"
+    env          = local.xdr_env
+  }
+
+  xdr_tags = jsonencode(local.xdr_tags_map)
```

Comment:
dont think you need to use `jsonencode` ? its simple list we are passing to the xdr_tags variable